evaluate.py

```diff
@@ -19,19 +20,41 @@ def main():
     parser.add_argument('--bootstrap-differences', action='store_true',
                         help='run bootstrap for differences')
     parser.add_argument('--output', type=Path, help='output results as json')
+    parser.add_argument('--parser', type=str, help='Specify a parser name to evaluate only that parser')
+    
     args = parser.parse_args()
     ground_truth = load_json(Path('ground-truth.json'))
     metrics_by_name = {}
-    for path in sorted(Path('output').glob('*.json')):
-        name = path.stem
+
+    if args.parser:
+        name = args.parser
+        path = Path('output') / f'{name}.json'
+        if not path.exists():
+            try:
+                extractor_module = importlib.import_module(f'extractors.run_{name}')
+                extractor_module.main()
+            except:
```

Comment:
I'd rather catch `Exception` here, e.g. see motivation in this (rejected) PEP https://peps.python.org/pep-0760/#motivation
```suggestion
            except Exception:
```